apps/admin-x-activitypub/src/views/Notifications/NotificationsV2.tsx

```
@@ -1,22 +1,350 @@
+                                                            <Skeleton className='w-full max-w-60' />
+                                                        </> :
+                                                        <NotificationGroupDescription group={group} />
+                                                    }
+                                                </div>
+                                                {(
+                                                    (group.type === 'reply' && group.inReplyTo) ||
+                                                    (group.type === 'like' && !group.post?.name && group.post?.content) ||
+                                                    (group.type === 'repost' && !group.post?.name && group.post?.content)
+                                                ) && (
+                                                    <div
+                                                        dangerouslySetInnerHTML={{__html: stripHtml(group.post?.content || '')}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)